framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2334,6 +2334,9 @@ public ParameterizedExecutableType methodFromUse(MethodInvocationTree tree) {
             addDefaultAnnotations(wildcard);
         }
 
+        List<AnnotatedTypeMirror> parameters =
```

Comment:
Can you add a comment here (and the same in `constructorFromUse`) what this does?
As this is `methodFromUse`, I'm not clear why we need this.